lib/github_fetcher.rb

```diff
@@ -28,13 +32,23 @@ def list_pull_requests
 
   def pull_requests_from_github
     repos.flat_map do |repo|
+      @repo_specific_alerts[repo] = @security_alert_handler.filter_security_alerts(repo) if @security_alert_handler
       fetch_pull_requests(repo).reject(&:draft)
     end
   end
 
+  def security_alerts_count
+    @security_alert_handler&.security_alerts_count
+  end
+
+  def github_api_errors
+    @security_alert_handler&.github_api_errors
+  end
```

Comment:
Nitpick: should we default to returning `0` if `@security_alert_handler` is undefined?